.github/workflows/build.yml

```
@@ -0,0 +1,35 @@
+name: Build
+
+on:
+  pull_request:
+    branches: [ main, master, develop ]
```

This workflow should be include for all branches 
```suggestion
    branches: [ '**' ]
```

---

Thanks, I have made changes. Please have a look once.

Thanks for your time.